gpt_index/readers/download.py

```diff
@@ -14,9 +15,51 @@
 
 from gpt_index.readers.base import BaseReader
 
-LOADER_HUB_URL = (
-    "https://raw.githubusercontent.com/emptycrown/loader-hub/main/loader_hub"
-)
+LLAMA_HUB_CONTENTS_URL = "https://api.github.com/repos/ahmetkca/llama-hub/contents"
+LOADER_HUB_PATH = "/loader_hub{path}"
+LOADER_HUB_URL = LLAMA_HUB_CONTENTS_URL + LOADER_HUB_PATH + "?ref=github-reader"
+
+modules_init_file_content = """\"\"\"
+Init file for modules directory.
+
+This file is automatically generated by the `gpt_index` library.
+Loaders are downloaded from the Loader Hub and stored in this directory.
+
+For example, if you want to download the `GithubRepositoryReader` loader,
+you can do the following:
+```python
+from gpt_index import download_loader
+download_loader("GithubRepositoryReader")
+
+from modules.github_repo import GithubClient, GithubRepositoryReader
+
+github_client = GithubClient(...)
+loader = GithubRepositoryReader(...)
+```
+\"\"\"
+"""
+
+
+def _get_file_content(path: str) -> str:
+    """
+    Get the content of a file from the GitHub REST API.
+
+    Decodes the content from base64-encoded string into a string using utf-8.
+
+    Args:
+        - path: The path of the file in the Loader Hub repo.
+
+    Returns:
+        The content of the file as a string.
+    """
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+    resp = requests.get(LOADER_HUB_URL.format(path=path), headers=headers).json()
+    content = resp["content"]
+    assert resp["encoding"] == "base64"
```

Comment:
for user-facing apps, let's `raise ValueError(...)` instead of doing `assert` calls